src/upstream_rbus.c

```diff
@@ -241,3 +244,87 @@ void eventReceiveHandler( rbusHandle_t rbus_Handle, rbusEvent_t const* event, rb
     }
 }
 #endif
+
+rbusError_t SendRbusEventCloudConnOnline()
```

Comment:
SendRbusEventCloudConnOnline - rename this to sendConnOnlineEvent() . check all places